kornia/augmentation/utils/helpers.py

```diff
@@ -3,6 +3,8 @@
 import torch
 from torch.distributions import Uniform, Beta
 
+from kornia.utils import _extract_device_dtype
```

Comment:
Is this redundant?

Reply:
No, it's not redundant. It's used by `_adapted_uniform` and `_adapted_beta`.